### PR TITLE
TF-2676, 2724 Fix creating identity with space only in name

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#2676 
#2724 

## Resolved

- [x] Trim identity name space
- [x] Add loading bar for list identities view 

## Demo 

- `#2676`

https://github.com/linagora/tmail-flutter/assets/80730648/93b90c7c-3be0-4753-b1e4-b8b96f3869ce

- `#2724`

https://github.com/linagora/tmail-flutter/assets/80730648/758accd4-5d92-46b6-80ba-7624294502a3

### CODE REVIEW

**Review by tddang-linagora — APPROVED** [review]

**Comment by hoangdat** [comment]
- [ ] Focus is in name, but screen is scroll down

**Comment by dab246** [comment]
> * [x]  Focus is in name, but screen is scroll down [[Android/IOS - WEB view only] Automatically scroll down when user click to edit one Profile Identity #2724](https://github.com/linagora/tmail-flutter/issues/2724)

## Demo


https://github.com/linagora/tmail-flutter/assets/80730648/623b2341-7387-478f-9fa4-0931e54f9949

**Review by tddang-linagora — APPROVED** [review]

**Review by tddang-linagora — DISMISSED** [review]

**Comment by hoangdat** [comment]
- please rebase @dab246

**Comment by dab246** [comment]
> * please rebase @dab246

Done. Please review again @hoangdat

**Comment by hoangdat** [comment]
- [ ] we lost button at the bottom of identity editor
image

File: lib/features/manage_account/presentation/profiles/identities/widgets/identity_loading_widget.dart [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return settingViewState.fold(
+      (failure) => identityViewState.fold(
+        (failure) => const SizedBox.shrink(),
+        (success) {
+          if (success is GetAllIdentitiesLoading || success is GetSessionLoading) {
+            return const CircleLoadingWidget(padding: EdgeInsets.all(16.0));
```
Comment (hoangdat) [inline lib/features/manage_account/presentation/profiles/identities/widgets/identity_loading_widget.dart:27]: how about we apply the same loading with `mark as read` or `get server settings`?

Comment (dab246) [reply lib/features/manage_account/presentation/profiles/identities/widgets/identity_loading_widget.dart:27]: > how about we apply the same loading with `mark as read` or `get server settings`?

Those 2 things are not relevant here.

**Comment by dab246** [comment]
> * [x]  we lost button at the bottom of identity editor
> 
> image

Fixed.

https://github.com/user-attachments/assets/0d6d17ec-1a3f-4a69-9d52-5aaa8d0c07ed

**Review by hoangdat — APPROVED** [review]

**Pull request merged**